README.md

```diff
@@ -91,16 +219,627 @@ impl DeserializeFromValue<MyError> for Name {
+| where_predicate     |  no   |  yes   | Let you add where clauses to the generated `DeserializeFromValue` implementation |
+
+#### Field attributes
+
+| features            | serde | deserr | note |
+|---------------------|-------|--------|------|
+| rename              |  yes  |  no    |      |
+| alias               |  yes  |  no    |      |
+| default             |  yes  |  yes   |      |
+| flatten             |  yes  |  no    | serde doesn't support flattening + denying unknown field |
+| skip                |  yes  |  yes   |      |
+| deserialize_with    |  yes  |  no    | But it's kinda emulated with `from` |
+| with                |  yes  |  no    |      |
+| borrow              |  yes  |  no    |      |
+| bound               |  yes  |  no    |      |
+| map                 |  no   |  yes   | Allows you to map the value **after** it was deserialized |
+| from                |  no   |  yes   | Deserialize this type from another type |
+| missing_field_error |  no   |  yes   | Allows you to return a custom error if this field is missing |
+| error               |  no   |  yes   | Specify the error type that should be used while deserializing this field |
+
+### Feature flags
+
+#### `serde-json`
+
+Import [`serde_json`](https://crates.io/crates/serde_json) and provide;
+- An implementation of `deserr::IntoValue` for `serde_json::Value` which make it easy to use both crate together.
+- A default implementation of the `JsonError` type that provide the best generic error messages possible.
+
+#### `serde-cs`
+Import [`serde-cs`](https://crates.io/crates/serde-cs) and provide;
+- An implementation of `DeserializeFromValue` for `serde_cs::CS<R>`.
+
+#### `actix-web`
+Import [`actix-web`](https://crates.io/crates/actix-web) and [`futures`](https://crates.io/crates/futures) and provide;
+- An implementation of a json actix-web extractor if used with the `serde-json` feature.
+- An implementation of `ResponseError` for the `JsonError` type if used with the `serde-json` feature.
+
+
+### FAQ
+
+#### But why?
+At meilisearch, we wanted to customize the error code we return when we fail
+the deserialization of a specific field.
+Some error messages were also not clear at all and impossible to edit.
+
+#### What about the maintenance?
+We're already using deserr in production at meilisearch; thus, I can guarantee you
+we'll maintain it for a long time.
+
+#### Where can I see more example of usage of this crate?
+Currently, you can read our examples in the `examples` directory of this repository.
+You can also look at our integration test; each attribute has a simple-to-read test.
+
+And obviously, you can read the code of meilisearch where deserr is used on all our
+routes.
+
+#### My question is not listed
+Please, if you think there is a bug in this lib or would like a new feature,
+open an issue or a discussion.
+If you would like to chat more directly with me, you can join us on discord
```

Comment:
first person here might be unclear. We should either specify, or use first person plural ("to chat more directly with us")